File: gcd/c#/gcd.cs
using System;

public class gcd
{
	public static int GCD(int a, int b)
	{
		if (b == 0)
			return a;
		else 
			return GCD(b, a % b);
	}

	static void Main(string[] args)
	{
		Console.WriteLine(GCD(36, 6));
	}
}

File: gcd/cpp/gcd.cpp
#include <bits/stdc++.h>
using namespace std;
// Iterative gcd function
int gcd(int m, int n)
{
    return __gcd(m,n);
}


int main()
{
    int m = 0;
    int n = 0;

    std::cout << "Please enter a number\n";
    std::cin >> m;

    std::cout << "Please enter another number\n";
    std::cin >> n;
    
    std::cout << "GCD of m and n: " << gcd(m, n) << '\n';

    return 0;
}
